pkg/quotaplugins/quota-forest/quota-manager/quota/core/treesnapshot.go

```diff
@@ -0,0 +1,197 @@
+/*
```

Comment:
Would it be possible to cover this struct with an unit test ?

Reply:
added unit test for TreeSnapshot